cbdr/cortex
# COR-738: Change Tag Input to be Full Width

**Purpose:**
A small PR to override the styles for the Bootstrap Tag Input we're using at the moment to more closely resemble the rest of the form.

> As a writer, I want the tag field to match the other fields in the blog post creation screen.

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/COR-738

**Changes:**
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**Screenshots**
* Before
<img width="493" alt="screen shot 2017-04-05 at 11 09 33 am" src="https://cloud.githubusercontent.com/assets/8419757/24715268/6e316256-19f0-11e7-940b-7a23e82394cf.png">

* After
<img width="610" alt="screen shot 2017-04-05 at 11 09 20 am" src="https://cloud.githubusercontent.com/assets/8419757/24715276/71d9fabc-19f0-11e7-9337-f26e488833f3.png">

**QA Links:**
C66 is down ATM

**How to Verify These Changes**
* Specific pages to visit
  * Create a new content item (doesn't matter, both have tags)

* Steps to take
  * Look at tag field.
  * Notice new changes

* Responsive considerations
  * N/A

**Relevant PRs/Dependencies:**
N/A

**Additional Information**
N/A
